e2e/connection_test.go

```diff
@@ -0,0 +1,21 @@
+package e2e
+
+import (
+	"flag"
+	"github.com/mysterium/node/tequilapi/client"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var host = flag.String("tequila.host", "localhost", "Specify tequila host for e2e tests")
+var port = flag.Int("tequila.port", 4050, "Specify tequila port for e2e tests")
+
+func TestClientConnectsToNode(t *testing.T) {
```

Comment:
Method name is misleading - it doesn't test that.

Reply:
Renamed.